stanford_news.module

```diff
@@ -35,32 +37,20 @@ function stanford_news_theme() {
  * Implements hook_preprocess_block().
  */
 function stanford_news_preprocess_block(&$variables) {
+
+  // Attach Library to the signup block wherever it goes.
   if (!empty($variables['elements']['#id'])) {
     if ($variables['elements']['#id'] == 'newslettersignup') {
       $variables['#attached']['library'][] = 'stanford_news/newsletter_signup';
     }
   }
+
+  // Attach library to news_list block.
   if (!empty($variables['elements']['content']['#cache']['contexts'])) {
     if (in_array('route.menu_active_trails:news-topics', $variables['elements']['content']['#cache']['contexts'])) {
       $variables['#attached']['library'][] = 'stanford_news/news_list';
     }
   }
-  $variables['attributes']['class'][] = Html::cleanCssIdentifier(_stanford_news_change_characters($variables['base_plugin_id']));
-  $variables['attributes']['class'][] = Html::cleanCssIdentifier(_stanford_news_change_characters($variables['derivative_plugin_id']));
```

Comment:
@boznik I've got to doublecheck on this but from what I could tell we aren't using the css classes from this code and the functionality is being duplicated in stanford_basic.

Reply:
@boznik assigned.